packages/core-manager/src/actions/info-resources.ts

```diff
@@ -10,8 +11,9 @@ export class Action implements Actions.Action {
     public async execute(params: any): Promise<any> {
         return {
             cpu: await this.prepareCpuData(),
-            mem: await this.prepareMemData(),
-            filesystems: await this.prepareFilesystemsData(),
+            ram: await this.prepareMemData(),
+            disks: await this.prepareFilesystemsData(),
+            installationDisk: await this.getProjectFilesystem(),
```

Comment:
If we are already adding this we can rename this to `disk` and drop the info about all disks because it's only relevant for us to know how much storage there is on the partition we are using.